docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -664,6 +670,246 @@ Delegates votes from the sender to `delegatee`.
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_governance::votes::VotesComponent;
+```
+Component that implements the {IVotes} interface and provides a flexible system for tracking and delegating voting power.
+
+By default, token balance does not account for voting power. This makes transfers cheaper. The downside is that it requires users to delegate to themselves in order to activate checkpoints and have their voting power tracked.
+
+NOTE: When using this module, your contract must implement the {VotingUnitsTrait}. For convenience, this is done automatically for `ERC20` and `ERC721` tokens.
+
+[.contract-index]
+.Voting Units Trait Implementations
+--
+.ERC20VotesImpl
+* xref:#VotesComponent-ERC20VotesImpl-get_voting_units[`++get_voting_units(self, account)++`]
+
+.ERC721VotesImpl
+* xref:#VotesComponent-ERC721VotesImpl-get_voting_units[`++get_voting_units(self, account)++`]
+--
+
+[.contract-index#VotesComponent-Embeddable-Impls]
+.Embeddable Implementations
+--
+[.sub-index#VotesComponent-Embeddable-Impls-VotesImpl]
+.VotesImpl
+* xref:#VotesComponent-get_votes[`++get_votes(self, account)++`]
+* xref:#VotesComponent-get_past_votes[`++get_past_votes(self, account, timepoint)++`]
+* xref:#VotesComponent-get_past_total_supply[`++get_past_total_supply(self, timepoint)++`]
+* xref:#VotesComponent-delegates[`++delegates(self, account)++`]
+* xref:#VotesComponent-delegate[`++delegate(self, delegatee)++`]
+* xref:#VotesComponent-delegate_by_sig[`++delegate_by_sig(self, delegator, delegatee, nonce, expiry, signature)++`]
+--
+
+[.contract-index]
+.Internal implementations
+--
+.InternalImpl
+* xref:#VotesComponent-_delegate[`++_delegate(self, account, delegatee)++`]
+* xref:#VotesComponent-move_delegate_votes[`++move_delegate_votes(self, from, to, amount)++`]
+* xref:#VotesComponent-transfer_voting_units[`++transfer_voting_units(self, from, to, amount)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#VotesComponent-DelegateChanged[`++DelegateChanged(delegator, from_delegate, to_delegate)++`]
+* xref:#VotesComponent-DelegateVotesChanged[`++DelegateVotesChanged(delegate, previous_votes, new_votes)++`]
+--
+
+[#VotesComponent-VotingUnitsTrait-Implementations]
+==== ERC20VotesImpl
+
+[.contract-item]
+[[VotesComponent-ERC20VotesImpl-get_voting_units]]
+==== `[.contract-item-name]#++get_voting_units++#++(self: @ComponentState<TContractState>, account: ContractAddress) → u256++` [.item-kind]#internal#
+
+Returns the number of voting units for a given account.
+
+This implementation is specific to ERC20 tokens, where the balance
+of tokens directly represents the number of voting units.
+
+NOTE: This implementation will work out of the box if the ERC20 component
+is implemented in the final contract.
+
+WARNING: This implementation assumes tokens map to voting units 1:1.
+Any deviation from this formula when transferring voting units (e.g. by using hooks)
+may compromise the internal vote accounting.
+
+==== ERC721VotesImpl
```

Comment:
We should add an ID here that follows the ERC20VotesImpl approach. Even if we go with something different than the above suggestion, this should still follow w/e pattern we use